gravity-forms/gw-advanced-merge-tags.php

```diff
@@ -506,6 +506,35 @@ public function handle_field_modifiers( $value, $input_id, $modifier, $field, $r
 						return rgar( $value_array, $index );
 					}
 					break;
+				case 'all_fields':
```

Comment:
I am open to different names for this like `all_dynamic_entries` or something? Curious what @spivurno thinks.